src/main/java/skylark/task/ToDo.java

```diff
@@ -0,0 +1,48 @@
+package skylark.task;
+
+/** Represents a ToDo Task. */
+public class ToDo extends Task {
```

Comment:
Nitpick but maybe rename this to Todo to be safe in conforming to coding standard?